types/index.esm.d.ts

```diff
@@ -2900,7 +2900,7 @@ export interface TickOptions {
    * Color of tick
    * @see Defaults.color
    */
-  color: Scriptable<Color, ScriptableScaleContext>;
+  color: Scriptable<Color, ScriptableScaleContext> | Color[];
```

Comment:
```suggestion
  color: Scriptable<Color, ScriptableScaleContext> | readonly Color[];
```

Reply:
@LeeLenaleee what's the benefit of adding `readonly` here?

It actually might be more annoying to work with a `readonly` array because when you're just defining the options in a mutable object, you're forced to go freeze the array like so:
```ts
ticks: {
  labels: Object.freeze([ 'blue', 'red' ])
}
```
while allowing just `string[]` wouldn't require this.

Reply:
I assume it is because if you change it after you create the chart and then update the chart it does not reflect that in the chart so you cant change the value after creating the chart

Reply:
It does not look like `readonly` requires freezing the array. However, we have an helper for this:

```suggestion
  color: ScriptableAndArray<Color, ScriptableScaleContext>;
```